core/logging.py

```diff
@@ -35,9 +36,10 @@ def emit(self, record: logging.LogRecord):
             message,
             resource=self.resource,
             labels={
-                "project_id": projectID,
+                "project_id": conf["viur.instance.project_id"],
                 "module_id": "default",
-                "version_id": appVersion if not isLocalDevelopmentServer else "dev_appserver",
+                "version_id": conf["viur.instance.app_version"] if not conf[
+                    "viur.instance.is_dev_server"] else "dev_appserver",
```

Comment:
This line break is very ugly.


I suggest sth. like this:
```suggestion
                "version_id": (conf["viur.instance.app_version"] 
                                         if not conf["viur.instance.is_dev_server"] 
                                         else "dev_appserver"),
```

Same for lines 15-16 above.